src/components/LinkText/LinkText.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { LinkTextProps } from './LinkText.model';
+import { Link } from '@nextui-org/link';
+import styles from './LinkText.module.scss';
+
+const LinkText = ({
+  label,
+  onClick,
+  rootClass,
+  underline = true,
+  className = '',
+  ...rest
+}: LinkTextProps) => {
+  return (
+    <Link
+      {...rest}
+      onClick={onClick}
+      className={`${styles.text} ${className} ${rootClass || ''}`}
```

Comment:
there are two props (className and rootClass) defined for same component,  any one of these should not required,